src/utils/dictionaries.ts

```diff
@@ -0,0 +1,16 @@
+export const HOST = "http://localhost:3001/";
+export const LOGInHost = HOST+ "login";
+export const RecruiterAddPost = HOST+ "recruiter/add";
+export const RegisterActive = HOST+ "login/register";
+export const CheckLogin = HOST+ "login/check";
+export const PasswordChange = HOST + "login/passchange";
+export const LogoutHost = HOST + "login/out";
+export const EmailChangeHost = HOST + "login/emailchange";
+export const StudentAddHost = HOST + "student/add";
+export const StudentRegisterHost = HOST + "student/register";
```

Comment:
Te nazwy są ekstremalnie losowe i wygląda to bardzo nieelegancko.
Z jednej strony `HOST`, potem jednak CamelCase. Niektóre mają postfix `Host` (niesłusznie, bo powinien być Url), a niektore nie mają go w ogóle. Zdecydowanie bym to zmienił, żeby było spójnie